astropy/time/core.py

```diff
@@ -1928,19 +1928,19 @@ def now(cls):
         method is called.
 
         .. note::
-            "Now" is determined using the `~datetime.datetime.utcnow`
+            "Now" is determined using the `~datetime.datetime.now`
             function, so its accuracy and precision is determined by that
             function.  Generally that means it is set by the accuracy of
-            your system clock.
+            your system clock. Timezone is set to UTC.
 
         Returns
         -------
         nowtime : :class:`~astropy.time.Time`
             A new `Time` object (or a subclass of `Time` if this is called from
             such a subclass) at the current time.
         """
-        # call `utcnow` immediately to be sure it's ASAP
-        dtnow = datetime.utcnow()
+        # call `now` immediately to be sure it's ASAP
+        dtnow = datetime.now(tz=UTC)
```

Comment:
comment: despite the fact that it is refactored to be a timezone aware datetime, class `Time` holds a naive datetime. This most probably breaks no API, but I fear it represent a somewhat limited refactoring.